### PR TITLE
A lot of changes : see description

### PR DESCRIPTION
**Pull request by izio38**
Just to track the feature.

* ~Introduce paletteId on `PedComponent` which seems to be useless ?~ *Removed*
* Fix a command permission bug.
* Allow command to be executed from Rcon.
* Ensure skin migrations are run.
* Handle save editor logic.
* Add a command to invoke the SkinEditor `/skin`.
* Handle future join when already having a skin on this particular player identity.
* Fix a serialization issue when having a serializable that has a serializable property.
* Add skin tones/faces based on parents, thanks @ArkSeyonet.
* Add `rpchat` module from @ArkSeyonet. Which allow admin to use `/me` and `/life`, for now it's more a PoC than other thing, bu we'll improve it.
* Refacto

### CODE REVIEW

**Comment by izio38** [comment]
I think we're good to go.

File: modules/__examples__/menu/client/module.lua [hunk 1]
```diff
@@ -63,5 +68,12 @@ module.onItemChanged = function(item, prop, val, index)
 end
 
 module.onItemClicked = function(item, index)
-  print('index', index)
+    if (item.name == "superSumbitButton") then
```
Comment (z4q) [inline modules/__examples__/menu/client/module.lua:71]: superSumbitButton -> superSubmitButton

Comment (SplashZK) [reply modules/__examples__/menu/client/module.lua:71]: in the items array has the same name, if change this line, change the name in the array.

Comment (izio38) [reply modules/__examples__/menu/client/module.lua:71]: Seems totally fair. Doing it right now.

**Pull request merged**